Mathlib/Analysis/Calculus/MeanValue.lean

```diff
@@ -8,9 +8,12 @@ import Mathlib.Analysis.Calculus.Deriv.Slope
 import Mathlib.Analysis.Calculus.Deriv.Mul
 import Mathlib.Analysis.Calculus.Deriv.Comp
 import Mathlib.Analysis.Calculus.LocalExtr.Rolle
-import Mathlib.Analysis.Convex.Normed
 import Mathlib.Analysis.RCLike.Basic
 import Mathlib.Topology.LocallyConstant.Basic
+import Mathlib.RingTheory.LocalRing.Basic
+import Mathlib.Topology.Instances.RealVectorSpace
+import Mathlib.Analysis.Normed.Group.AddTorsor
+import Mathlib.Analysis.Convex.Normed.Topology
```

Comment:
Can you `#min_imports` this? I doubt you need `LocalRing` here

Reply:
Done.